repose-aggregator/core/core-lib/src/test/scala/org/openrepose/core/spring/CoreSpringProviderTest.scala

```diff
@@ -102,4 +102,22 @@ class CoreSpringProviderTest extends FunSpec with Matchers with TestFilterBundle
     }
   }
 
+  describe("Repose Spring Properties") {
```

Comment:
when we talk about them, we call these timebomb tests. Could you add the word time bomb in the title or comment so that if we do a search for timebomb it'll pop up?